app/src/forms/files/middleware/upload.js

```diff
@@ -0,0 +1,103 @@
+const bytes = require('bytes');
+const fs = require('fs-extra');
+const multer = require('multer');
+const os = require('os');
+
+const Problem = require('api-problem');
+
+let uploader = undefined;
+let storage = undefined;
+let maxFileSize = bytes.parse('25MB');
+let maxFileCount = 1;
+
+module.exports.fileUpload = {
+
+  init(options) {
+    const fileUploadsDir = (options && options.dir) || process.env.FILE_UPLOADS_DIR || fs.realpathSync(os.tmpdir());
```

Comment:
Should we be using the config library instead of directly polling env variables? (Also applies to lines 22, 29 and 36)